File: HarmonicArchive-backend/models/MusicSheetInstrument.js
const { DataTypes } = require('sequelize');
const sequelize = require('../data/db');

const MusicSheetInstrument = sequelize.define('MusicSheetInstrument', {
  musicSheetId: {
    type: DataTypes.INTEGER,
    primaryKey: true,
    references: {
      model: 'MusicSheets',
      key: 'id'
    }
  },
  instrumentId: {
    type: DataTypes.INTEGER,
    primaryKey: true,
    references: {
      model: 'Instruments',
      key: 'id'
    }
  }
});

module.exports = MusicSheetInstrument;

File: HarmonicArchiveBackend/Models/MusicSheetInstrument.cs
namespace HarmonicArchiveBackend.Models
{
    public class MusicSheetInstrument
    {
        public int MusicSheetId { get; set; }
        public MusicSheet MusicSheet { get; set; }

        public int InstrumentId { get; set; }
        public Instrument Instrument { get; set; }
    }
}
